src/components/display/LoadingRow.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react"
```

Comment:
I think we should rename this component to `SkeletonTableRow`, I think it makes more sense in the context of a standalone library